src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay/advanced-features.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "Advanced features"
+metaDescription: "Manually record session replays, disable automatic session replay tracking and use nerd graph to configure replay settings."
+freshnessValidatedDate: 2024-12-19
+---
+
+## Manually record session replays [#manual-replays]
+
+If needed, you can programmatically `force` a session replay to start or stop recording using API methods. Here are some use cases:
+
+* Allow users to opt in for interaction recording. For example, if you have a pop-up asking "Allow this session to be recorded for performance analysis?" and a user consents:
+  1. Call `newrelic.recordReplay()` to initiate recording.
+  2. Once the session is complete or consent is withdrawn, use `newrelic.pauseReplay()` to stop capturing data.
```

Comment:
After instead of once
<reason>, <action>

Reply:
Updated